src/graph/DeleteEdgesExecutor.cpp

```diff
@@ -19,34 +19,34 @@ DeleteEdgesExecutor::DeleteEdgesExecutor(Sentence *sentence,
 }
 
 Status DeleteEdgesExecutor::prepare() {
-    Status status;
-    do {
-        status = checkIfGraphSpaceChosen();
-        if (!status.ok()) {
-            break;
-        }
-        spaceId_ = ectx()->rctx()->session()->space();
-        expCtx_ = std::make_unique<ExpressionContext>();
-        expCtx_->setSpace(spaceId_);
-        expCtx_->setStorageClient(ectx()->getStorageClient());
-
-        auto edgeStatus = ectx()->schemaManager()->toEdgeType(spaceId_, *sentence_->edge());
-        if (!edgeStatus.ok()) {
-            status = edgeStatus.status();
-            break;
-        }
+    spaceId_ = ectx()->rctx()->session()->space();
```

Comment:
What's reason move the `checkIfGraphSpaceChosen` to `execute()`?

Reply:
For example : use space; delete index ...